open-problems/brief-problem-statements.md

```diff
@@ -0,0 +1,15 @@
+The most important problems/technologies relevant to Protocol Labs that will/should exist 5-10 years from now:
+
+- certifiable archives (.car)
```

Comment:
Shouldn't this one be on the ipfs/research?

Reply:
No harm in adding it to the list in IPFS/research.  As it came up in the conversation, it was treated as part of libp2p (I think to put it in the lowest library that would use it), but I think you could make a good argument for it being in IPFS, IPLD, or probably even multiformats.